woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -517,7 +522,9 @@ echo "WAIT MODULES LOADED"
 #previous module loading may not have completed...
 echo -n "Waiting for modules to complete loading..." >/dev/console
 PCMCIABUILTIN='no'
-if [ "`modinfo yenta_socket 2>/dev/null`" = "" ];then   #Marvmod shut this up.
+if modinfo yenta_socket | grep -m 1 -q '(builtin)';then
```

Comment:
`yenta_socket` can be builtin `lspci` although the output of `modinfo yenta_socket` is not empty, so the `lspci` can be skipped